dogs.html

```diff
@@ -0,0 +1,114 @@
+        <h1>ADOPTADOG</h1>
+        <p>Seattle's premiere dog adoption agency</p>
+      </div>
+      <div class="checkout">
+        <h5><!--shopping cart fontawesome--> $1234.56</h5>
+        <h5>Checkout</h5>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="Dogs.html">Dogs</a></li>
+          <li><a href="Blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="main">
+        <!-- <img src="images/eric-ward-610868-unsplash.jpg" alt="Eric Ward With Dog"> -->
+        <h2>Here are all of our Dogs!</h2>
+        <!-- <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Libero omnis ad sit a temporibus ipsum beatae eius dicta eaque aliquid officia consequatur sint illo placeat, perferendis quisquam ducimus nisi fuga!</p> -->
+      </div>
+      <div class="item1">
```

Comment:
You should use the same css class for all of the dogs instead of a different one for each